src/components/graph/Graph.jsx

```diff
@@ -310,10 +317,10 @@ export default class Graph extends React.Component {
 
     d3SelectAll(`#${this.state.id}-${CONST.GRAPH_CONTAINER_ID}`).attr("transform", transform);
 
-    this.state.config.panAndZoom && this.setState({ transform: transform.k });
+    this.setState({ transform: transform });
```

Comment:
If you want to assign a variable to a property of the same name, you can use the [property shorthand](http://es6-features.org/#PropertyShorthand) syntax to avoid repeating `transform`: 

```
this.setState({ transform });
```